test/pytest/test_utils.py

```diff
@@ -59,6 +59,7 @@ def start_torchserve(
     no_config_snapshots=False,
     gen_mar=True,
     plugin_folder=None,
+    token=False,
```

Comment:
lets make this less an a logic puzzle and name is disable_token so we do not need to invert below.